roles/chameleon_usage/templates/site.yaml.j2

```diff
@@ -0,0 +1,7 @@
+---
+username: "{{ chameleon_usage_site_user }}"
+password: "{{ chameleon_usage_site_password }}"
+
+maintenance_project_id: {{ maintenance_project_id }}
```

Comment:
I missed where this is set - is it something the operator has to set in site-configs? It would be nice if this was automatic somehow (or, used a project name rather than ID, making it generic across sites.)

Reply:
by convention, I believe we've been using `openstack` for the admin project, and `chameleon` for the devops project, but this doesn't extend well to associate sites.

I agree that some automation would be nice, both for creation of the projects (if local), and making it generic.
I'm hesitant to add too many settings that require specific configuration at every site.